crates/bevy_text/src/text.rs

```diff
@@ -6,6 +6,8 @@ use bevy_utils::default;
 use serde::{Deserialize, Serialize};
 
 use crate::{Font, DEFAULT_FONT_HANDLE};
+// TODO: reexport cosmic_text and these types in the prelude
+pub use cosmic_text::{FamilyOwned, Stretch, Style, Weight};
```

Comment:
Re-exporting `Style` as such will create a nasty clash with `bevy_ui`'s style component.

Reply:
Would this work?
```rs
pub use cosmic_text::{
    FamilyOwned as FontFamily,
    Stretch as FontStretch,
    Style as FontStyle,
    Weight as FontWeight
};
```